### PR TITLE
Feature/header - Implement header according to UI specifications

### PR DESCRIPTION
**Pull request by ramindu-Nimex**
full screen
I-pad-mini
I-phone 14 Pro max
I-phone se

### CODE REVIEW

File: public/Facebook.png
Comment (sathninduk) [inline public/Facebook.png:1]: Reorganize and refactor the file path (ex: public -> images -> logos)

**Review by sathninduk — CHANGES_REQUESTED** [review]

File: components/Header/index.jsx [hunk 1]
```diff
@@ -1,66 +1,170 @@
...
+            <ul className="flex justify-around gap-12 items-center">
+              {navItems.map((item, index) => (
+                <li key={index}>
+                  {item.link ? (
+                    <Link href={item.link} className="text-lg font-semibold">
+                      {item.name}
+                    </Link>
+                  ) : (
+                    <Scroll
+                      className="text-lg font-semibold"
```
Comment (Randula98) [inline components/Header/index.jsx:65]: When hovering the text element, the cursor should be a pointer. Use tailwindcss inline styles to add that.

File: components/Header/index.jsx [hunk 2]
```diff
@@ -1,66 +1,170 @@
...
+              alt="menu"
+              className="cursor-pointer w-6 h-6 sm:w-8 sm:h-8"
+              onClick={handleNav}
+            />
+          </div>
+          {/* social media */}
+          <div className="flex items-center justify-center">
+            {socialMedia.map((social, index) => (
+              <Link href={social.link} key={index}>
+                <Image
```
Comment (Randula98) [inline components/Header/index.jsx:94]: When hovering the text element, the cursor should be a pointer. Use tailwindcss inline styles to add that.

File: components/Header/index.jsx [hunk 3]
```diff
@@ -1,66 +1,170 @@
 "use client";
 import { Link as Scroll } from "react-scroll";
 import Link from "next/link";
+import logomain from "../../public/images/logos/logomain.png";
```
Comment (Randula98) [inline components/Header/index.jsx:4]: Please review on image import syntax on next.js

File: components/Header/index.jsx [hunk 4]
```diff
@@ -1,66 +1,170 @@
...
+import Facebook from "../../public/images/logos/Facebook.png";
+import Instagram from "../../public/images/logos/Instagram.png";
+import Twitter from "../../public/images/logos/Twitter.png";
+import GitHub from "../../public/images/logos/GitHub.png";
+import Image from "next/image";
+import { AiOutlineClose } from "react-icons/ai";
+import { useState } from "react";
+
+const socialMedia = [
+  { src: Facebook, alt: "Facebook", link: "/" },
```
Comment (Randula98) [inline components/Header/index.jsx:15]: for example. 
/images/logos/Facebook.png

change the rest as nessasary

**Review by Randula98 — CHANGES_REQUESTED** [review]

**Review by Randula98 — APPROVED** [review]
LGTM

**Review by sathninduk — APPROVED** [review]
LGTM

**Pull request merged**